File: app/src/main/java/mo/wall/org/dialog/MyDialogActivity.java
package mo.wall.org.dialog;

import android.content.Intent;
import android.os.Bundle;
import android.os.Message;
import android.view.View;

import androidx.fragment.app.Fragment;

import org.wall.mo.base.activity.AbsWithV4FragmentActivity;

import mo.wall.org.R;
import mo.wall.org.databinding.ActivityMyDialogBinding;

public class MyDialogActivity extends AbsWithV4FragmentActivity<ActivityMyDialogBinding, MyDialogAcceptPar> {

    @Override
    public int getContainerViewId() {
        return R.id.container;
    }


    @Override
    public int getLayoutId() {
        return R.layout.activity_my_dialog;
    }


    @Override
    public int getTopBarTitleViewId() {
        return R.id.tvTopBarTitle;
    }

    @Override
    public int getTopBarBackViewId() {
        return R.id.tvTopBarLeftBack;
    }

    @Override
    public Fragment createFragment() {
        Intent intent = getIntent();
        Bundle extras = null;
        if (intent != null) {
            extras = intent.getExtras();
        }
        return MyDialogFragment.newInstance(extras);
    }

    @Override
    public void loadIntentData(Intent intent) {

    }

    @Override
    public void handleSubMessage(Message msg) {

    }

    @Override
    public void onClick(View v) {

    }

    @Override
    public void showShortToast(String msg) {

    }

    @Override
    public void showLongToast(String msg) {

    }

    @Override
    public void showDialog(String msg) {

    }

    @Override
    public void hideDialog() {

    }
}

File: app/src/main/java/mo/wall/org/dialog/MyDialogActivity.kt
package mo.wall.org.dialog

import android.content.Intent
import android.os.Bundle
import android.os.Message
import android.view.View

import androidx.fragment.app.Fragment

import org.wall.mo.base.activity.AbsWithV4FragmentActivity

import mo.wall.org.R
import mo.wall.org.databinding.ActivityMyDialogBinding

class MyDialogActivity : AbsWithV4FragmentActivity<ActivityMyDialogBinding, MyDialogAcceptPar>() {

    override fun getContainerViewId(): Int {
        return R.id.container
    }


    override fun getLayoutId(): Int {
        return R.layout.activity_my_dialog
    }


    override fun getTopBarTitleViewId(): Int {
        return R.id.tvTopBarTitle
    }

    override fun getTopBarBackViewId(): Int {
        return R.id.tvTopBarLeftBack
    }

    override fun createFragment(): Fragment? {
        val intent = intent
        var extras: Bundle? = null
        if (intent != null) {
            extras = intent.extras
        }
//        val newInstance = MyDialogFragment.newInstance(extras)

        val newInstance = MyDialogFragment().apply {
            arguments = extras
        }
        return newInstance
    }

    override fun loadIntentData(intent: Intent) {

    }

    override fun handleSubMessage(msg: Message) {

    }

    override fun onClick(v: View) {

    }

    override fun showShortToast(msg: String) {

    }

    override fun showLongToast(msg: String) {

    }

    override fun showDialog(msg: String) {

    }

    override fun hideDialog() {

    }

    override fun showInfoDialog(msg: String?) {
    }

    override fun statusLoadingView() {
    }

    override fun statusNetWorkView() {
    }

    override fun statusErrorView(type: Int, msg: String?) {
    }

    override fun statusContentView() {
    }
}
